telegram/_bot.py

```diff
@@ -3466,6 +3470,10 @@ async def edit_message_caption(
         """
         Use this method to edit captions of messages.
 
+        Note:
+            Please note, that it is currently only possible to edit messages without
```

Comment:
```suggestion
            It is currently only possible to edit messages without
```

to keep it unified :)